### ISSUE TITLE
restore fails due to schema creation issues

### ISSUE THREAD

**Issue by moodgorning** [timescale/timescaledb-backup#46]
Hello, we are trying to migrate our pg12  TSDB 2.4 DB to a new cluster running on PG13 & TSDB 2.4. I am currently trying to use the ts-dump & ts-restore apps provided from you folks but am running into some issues:
I can pull a dump with ts-dump without problems, but when I try to restore Into a blank DB it fails complaining that the schema it want’s to install TSDB into doesn’t exist.
If I manually create the schema in the new DB before I run ts-restore, it will startup and do the pre-data part, but fail with
pg_restore run failed in pre-data section: cmd.Run() failed with 'exit status 1'
In the output, I can see that pg_restore also tried to create the schema which produced an error since the schema is already created by me. I am not sure if that is the error that causes the whole thing to crash, but it’s the only thing I can see.
To summarise:
If I run against a vanilla DB ts-restore complains about the schema not being there
If I run against a DB that has the schema it creates an error because it tries to recreate it. Any pointers will be highly appreciated
Versino info:
SourceDB: PG12 + TSDB 2.4
Target DB: PG13 + TSDB 2.4 running on a HA kubernetes custer with Patroni, following the provided Timsecale Helm charts
